bin/pocs_shell

```diff
@@ -156,11 +156,20 @@ simulate is a space-separated list of hardware to simulate.
 Hardware names: {}   (or all for all hardware)'''.format(
             ','.join(hardware.get_all_names())))
 
+    def complete_setup_pocs(self, text, line, begidx, endidx):
+        """Provide completions for simulator names."""
+        names = ['all'] + hardware.get_all_names()
+        return [name for name in names if name.startswith(text)]
+
     def do_reset_pocs(self, *arg):
+        """Discards the POCS instance.
+
+        Does NOT park the mount, nor execute power_down.
+        """
         self.pocs = None
 
     def do_run_pocs(self, *arg):
-        """ Make POCS `run` in a separate process
+        """Make POCS `run` in a separate process
 
         Note:
             Run with `--simulator=all` for simulator
```

Comment:
Should update this comment to reflect how `simulator` is used.

Reply:
I don't know what to update it with. The comment looks wrong, but I've not executed this command enough to know what to update this with... except the missing period at the end! ;-)

Reply:
Back in https://github.com/panoptes/POCS/commit/3cf2831b0dda9ecf3b8d70ab64069789d06a4c94#diff-f081d3c1efc3be3fbd0aa696e79e797a commit you had switched it do a space-separated list and without the `--simulator=`. The `Note:` here refers to commands to pass to `setup_pocs`, which is probably confusing.